docs/en/siem/field-ref.asciidoc

```diff
@@ -0,0 +1,265 @@
+* `auditd.summary.object.type` (Auditbeat field)
+* `auditd.summary.sequence` (Auditbeat field)
+* `destination.bytes`
+* `destination.geo.city_name`
+* `destination.geo.continent_name`
+* `destination.geo.country_iso_code`
+* `destination.geo.country_name`
+* `destination.geo.region_iso_code`
+* `destination.geo.region_name`
+* `destination.ip`
+* `destination.packets`
+* `destination.port`
+* `dns.question.name`
+* `dns.question.type`
+* `dns.resolved_ip`
+* `dns.response_code`
+* `event.action`
+* `event.category`
+* `event.code`
+* `event.created`
+* `event.dataset`
+* `event.duration`
+* `event.end`
+* `event.hash`
+* `event.id`
+* `event.kind`
+* `event.module`
+* `event.original`
+* `event.outcome`
+* `event.risk_score_norm`
+* `event.risk_score`
+* `event.severity`
+* `event.start`
+* `event.timezone`
+* `event.type`
+* `file.ctime`
+* `file.device`
+* `file.extension`
+* `file.gid`
+* `file.group`
+* `file.inode`
+* `file.mode`
+* `file.mtime`
+* `file.name`
+* `file.owner`
+* `file.path`
+* `file.size`
+* `file.target_path`
+* `file.type`
+* `file.uid`
+* `host.id`
+* `host.ip`
+* `http.request.body.bytes`
+* `http.request.body.content`
+* `http.request.method`
+* `http.request.referrer`
+* `http.response.body.bytes`
+* `http.response.body.content`
+* `http.response.status_code`
+* `http.version`
+* `message`
+* `network.bytes`
+* `network.community_id`
+* `network.direction`
+* `network.packets`
+* `network.protocol`
+* `network.transport`
+* `process.args`
+* `process.executable`
+* `process.hash.md5`
+* `process.hash.sha1`
+* `process.hash.sha256`
+* `process.name`
+* `process.pid`
+* `process.ppid`
+* `process.title`
+* `process.working_directory`
+* `rule.reference`
+* `source.bytes`
+* `source.geo.city_name`
+* `source.geo.continent_name`
+* `source.geo.country_iso_code`
+* `source.geo.country_name`
+* `source.geo.region_iso_code`
+* `source.geo.region_name`
+* `source.ip`
+* `source.packets`
+* `source.port`
+* `suricata.eve.alert.signature_id` {filebeat-ref}/exported-fields-suricata.html[(Filebeat field)]
+* `suricata.eve.alert.signature` (Filebeat field)
+* `suricata.eve.flow_id` (Filebeat field)
+* `suricata.eve.proto` (Filebeat field)
+* `system.audit.package.arch` {auditbeat-ref}/exported-fields-system.html[(Auditbeat field)]
+* `system.audit.package.entity_id` (Auditbeat field)
+* `system.audit.package.name` (Auditbeat field)
+* `system.audit.package.size` (Auditbeat field)
+* `system.audit.package.summary` (Auditbeat field)
+* `system.audit.package.version` (Auditbeat field)
+* `system.auth.ssh.method` {filebeat-ref}/exported-fields-system.html[(Filebeat field)]
+* `system.auth.ssh.signature` (Filebeat field)
+// Not documented since v7.3:
+// * `tls.client_certificate.fingerprint.sha1` {packetbeat-ref}/exported-fields-tls_detailed.html[(Packetbeat field)]
+// * `tls.fingerprints.ja3.hash` (Packetbeat field)
+// * `tls.server_certificate.fingerprint.sha1` (Packetbeat field)
```

Comment:
@MikePaquette @XavierM 
These fields are in the code but have not been documented in the filebeats module since 7.3: https://www.elastic.co/guide/en/beats/packetbeat/7.3/exported-fields-tls.html

Reply:
I think we should remove it, let me make a note of that